### DISCUSSION TITLE
Shorts shown in the feed are removed by the general ads patch

### DISCUSSION THREAD

**Discussion by Divivats** [ReVanced/revanced-discussions#289] [Question] [answered]
There are shorts which are shown in feed , but the general ads patch remove them as well.
I also posted a discussion about it before but the developer thought i was talking about the shorts button down below so this time i am posting a screenshot with it to tell what i mean. The marked shorts in feed(screenshot) are removed by the general ads patch.
![IMG_20220708_010621](https://user-images.githubusercontent.com/108374606/177857909-8d8f2368-6008-47bf-8fc4-b9b09b7e944a.jpg)

**Comment by ghost** [comment]
This is a litho patch that can will be disabled in future, after that patch to show settings will be ready.

Antyway...the patch in question is the following:

```
if (SettingsEnum.ADREMOVER_SHORTS_SHELF_BOOLEAN.getBoolean()) {
          blockList.add("shorts_shelf");
}
```

**Reply by TheJeterLP** [reply]
> This is a litho patch that can will be disabled in future, after that patch to show settings will be ready.
> 
> Antyway...the patch in question is the following:
> 
> ```
> if (SettingsEnum.ADREMOVER_SHORTS_SHELF_BOOLEAN.getBoolean()) {
>           blockList.add("shorts_shelf");
> }
> ```

Yes I know that already

**Reply by ghost** [reply]
Yeah...I was telling that to the user. 👍

**Comment by TheJeterLP** [comment, marked as answer]
So this can be disabled when the settings are implemented, for now theres no way of just showing those

**Reply by oSumAtrIX** [reply]
A protobuf blocking dependency patch will be created so each patch can insert their own strings such as the hide shorts patch which will insert its shorts string. This way modularity will be guaranteed and blockings won't happen when not required and the code will be in its respective patch.

**Reply by TheJeterLP** [reply]
Yes that would be a way to do it
